src/common/mockFactory.ts

```diff
@@ -53,14 +53,40 @@ export default class MockFactory {
         };
     }
 
+    /**
+     * @name Create Video Test Asset
+     * @description Creates fake video IAsset
+     * @param name Name of asset
+     * @param assetState State of asset
+     */
+    public static createVideoTestAsset(name: string, assetState: AssetState = AssetState.NotVisited): IAsset {
+        return {
+            id: `videoasset-${name}`,
+            format: "mp4",
+            name: `Video Asset ${name}`,
+            path: `C:\\Desktop\\videoasset${name}.mp4`,
+            state: assetState,
+            type: AssetType.Video,
+            size: {
+                width: 800,
+                height: 600,
+            },
+        };
+    }
+
     /**
      * Creates array of fake IAsset
      * @param count Number of assets to create
      */
     public static createTestAssets(count: number = 10): IAsset[] {
         const assets: IAsset[] = [];
-        for (let i = 1; i <= count; i++) {
-            assets.push(MockFactory.createTestAsset(i.toString()));
+        for (let i = 1; i <= count / 2; i++) {
+            assets.push(MockFactory.createVideoTestAsset(i.toString()));
+        }
+        if (count > 1) {
+            for (let i = 1; i <= count / 2; i++) {
+                assets.push(MockFactory.createTestAsset(i.toString()));
+            }
```

Comment:
why the use of 2 for loop and a conditional? Is there any way we could simplify this?

Reply:
ie, I could have done this all in one loop, such as:
```javascript
for (let i = 1; i <= count; i++) {
   assets.push((i % 2 == 0) ? MockFctory.createTestAsset(i.toString()) | MockFactory.createVideoTestAsset(i.toString())));
}
```
But I think the explicit version is more clear.